oci/private/tarball.sh.tpl

```diff
@@ -5,36 +5,120 @@ readonly STAGING_DIR=$(mktemp -d)
 readonly YQ="{{yq}}"
 readonly IMAGE_DIR="{{image_dir}}"
 readonly BLOBS_DIR="${STAGING_DIR}/blobs"
-readonly TARBALL_PATH="{{tarball_path}}"
+readonly TARBALL_PATH="$(pwd)/{{tarball_path}}"
```

Comment:
could you please explain why this change is needed?

Reply:
Because we're now globbing all the files for the `tar` invocation at the bottom (so that we can pick up whichever of `index.json` or `manifest.json`), to avoid needing to go through hoops to strip prefix directories, I figured it'd just be easier to `cd` into `${STAGING_DIR}` before invoking `tar` (rather than using `-C`), but that requires the destination being an absolute path

Reply:
I see. Can we revert this change as well? There’s a new hermetic toolchain coming up so we won’t need any of these changes and I’d like to keep it same to make it easier for bisecting in case something breaks.

Reply:
Done - expanded out the glob below so we don't need to absolutify the path here.